crates/anvil-polkadot/Cargo.toml

```diff
@@ -20,7 +20,7 @@ path = "bin/main.rs"
 [dependencies]
 # foundry internal
 substrate-runtime = { path = "substrate-runtime" }
-polkadot-sdk = { version = "2507.1.0", default-features = false, features = [
+polkadot-sdk = { git = "https://github.com/paritytech/polkadot-sdk.git", branch = "acihodaru/foundry-polkadot", default-features = false, features = [
```

Comment:
I guess you do this because your branch has this PR: https://github.com/paritytech/polkadot-sdk/pull/9701/files, while upstream doesn't. Wouldn't the same work if we backport the PR into unstable2507 and use it here instead? I know we try to introduce polkadot-sdk/master git dependency in a different PR, but seems like a low hanging fruit to make it based on `unstable2507` (assuming backporting is easy), which should give us some ability to backport relevant things we merge in master in the future, and use them in anvil-polkadot. 

@AndreiEres  - how far are we from using `polkadot-sdk/master` as a git dependency for polkadot-sdk crate here, given [#43](https://github.com/paritytech/foundry-compilers-polkadot/pull/43)  & #284 ?

Reply:
> @AndreiEres - how far are we from using polkadot-sdk/master as a git dependency for polkadot-sdk crate here, given https://github.com/paritytech/foundry-compilers-polkadot/pull/43 & https://github.com/paritytech/foundry-polkadot/pull/284 ?

The needed PRs were merged in master (there were other PRs, by Pavlo). We just need to rebase and fix conflicts